.dependabot/config.yml

```diff
@@ -6,10 +6,10 @@ update_configs:
     update_schedule: "live"
     version_requirement_updates: "increase_versions"
     ignored_updates:
-      # ui-core supports 16.3 of react, so ignore all updates
+      # ui-core supports 16.8 of react, so ignore all updates
       - match:
           dependency_name: "react"
-          version_requirement: ">=16.4.x"
+          version_requirement: ">=16.9.x"
       - match:
           dependency_name: "react-dom"
-          version_requirement: ">=16.4.x"
+          version_requirement: ">=16.9.x"
```

Comment:
@varl why version_requirement: ">=16.9.x" version 16.9? and not 16.8?

Reply:
I know this has been discussed on Slack, but I am still confused:
1. If this is a valid configuration key, should't we flip the comparison operator to `<=16.9.x`
2. Going through the dependabot docs page it seems to me that this key is not valid. We are using `version_requirement` but in [the docs](https://dependabot.com/docs/config-file/) I can only see a reference to `version_requirement_updates`

Are we sure about this?

Reply:
The top level key is `ignored_updates`: https://dependabot.com/docs/config-file/#ignored_updates

Their example:

```yaml
version: 1
update_configs:
  - package_manager: "javascript"
    directory: "/"
    update_schedule: "daily"
    ignored_updates:
      - match:
          dependency_name: "express"
          # The version_requirement specifies the versions to ignore.
          # The range format is specific to the package manager
          # (e.g., ^1.0.0 for JS, or ~> 2.0 for Ruby).
          version_requirement: "4.x"
      - match:
          # Wildcards match unlimited arbitrary characters (or none)
          dependency_name: "aws*"
```

The logic is that since we want 16.8, we want to **ignore** 16.9.x, but still allow updates for any patch releases for 16.8.x.

Reply:
aha gotcha